R/pkg/inst/tests/testthat/test_sparkSQL.R

```diff
@@ -2897,6 +2898,27 @@ test_that("Collect on DataFrame when NAs exists at the top of a timestamp column
   expect_equal(class(ldf3$col3), c("POSIXct", "POSIXt"))
 })
 
+compare_list <- function(list1, list2) {
+  # get testthat to show the diff by first making the 2 lists equal in length
+  expect_equal(length(list1), length(list2))
+  l <- max(length(list1), length(list2))
```

Comment:
The lengths should be equal if we get to this line ? Or am I missing something ?